macos/FluentUIUnitTest/AvatarViewTests.swift

```diff
@@ -140,20 +140,20 @@ class AvatarViewTests: XCTestCase {
 
 	func testHashAlgorithm () {
 		var colorSet = AvatarView.getInitialsColorSet(fromPrimaryText: "", secondaryText: "Annie.Lindqvist@example.com")
```

Comment:
Optional - could add some of the automated tests that FURN has just to double check FURN/FUA are aligned (esp since it looks like these tests are all mostly based off email instead of name)

<img width="702" alt="Screenshot 2022-11-01 at 10 18 46 AM" src="https://user-images.githubusercontent.com/78454019/199296474-1ecb4464-f2c7-4db5-a037-a80a0d9feaf1.png">

Reply:
Is the Id supposed to be idForColor?

Reply:
Yes

Reply:
I think it's probably fine to omit those test cases since FUA doesn't really have an equivalent of "idForColor", unless we decide the FUA Avatar API should change to match FURN (or vice versa). I can add the FUA test cases of email to FURN, I think that would make sense. If anyone else has thoughts on what test cases should be on FUA/FURN feel free to chime in